linux/9.3.x/sitecore-xp-solr/boot.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -m
+set -e
+
+cleanDir=$1
+dataDir=$2
+
+# prepare data
+if [ ! -f "$dataDir/solr.xml" ]; then
+    echo "### No solr.xml found in '$dataDir', seeding..."
+
+    cp -R --verbose $cleanDir/. $dataDir/
+
+    echo "### Done seeding."
+else
+    echo "### Done, existing data found in '$dataDir'..."
+
+    # cleanup old lock files
+    find $dataDir/. -name write.lock -print0 | xargs -0 rm -rf
+fi
+
+asyncRun() {
```

Comment:
This function should be removed, not called anymore... (should also be removed from 9.3 XM solr)

Reply:
removed, also from xm